10/mysql-tomcat/xwiki/docker-entrypoint.sh

```diff
@@ -104,6 +109,17 @@ function restoreConfigurationFile() {
   fi
 }
 
+function enableClustering() {
+  echo 'Setting clustering...'
+  xwiki_set_properties 'observation.remote.enabled' 'true'
+  if [ $CLUSTER_CHANNEL ]; then
+    echo "Setting cluster channel to $CLUSTER_CHANNEL"
+    xwiki_replace_example 'observation.remote.channels' "$CLUSTER_CHANNEL"
+  else
+    xwiki_replace_example 'observation.remote.channels' 'udp'
```

Comment:
Why is it needed to replace an example in the config file? This feels weird :)

Reply:
In xwiki.properties there is no field of
"# observation.remote.channels"
So I didn't find anything to replace. So I replaced the Example